lib/tasks/allrecipe-recipe_category_relations.rake

```diff
@@ -0,0 +1,72 @@
+task :recipe_category_relations => :environment do
+  require 'open-uri'
+  require 'nokogiri'
+  categories = Category.all
+  categories.each do |category|
+    @tries = 0
+    begin
+      # 0 = have not started, 1 = started but not finished, 2 = completed
+      if category.scrape_category_status == 0 || category.scrape_category_status == 1
+        category.update(scrape_category_status: 1)
+        url = category.sub_category_url
+        puts url
+        browser = open(url).read
+        html_doc = Nokogiri::HTML(browser)
+
+        recipe_amounts = html_doc.css('div > div.hubs-wrapper.fl-left > div.search-tools > p:nth-child(2).searchResultsCount.results > span:nth-child(1).emphasized').text.gsub(',', '').to_i
+
+        pages_to_loop = (recipe_amounts / 20.0).ceil
+        for i in 1..pages_to_loop do
+          puts "page " + i.to_s
+          get_recipe_basic_info(category.sub_category_url + "&page=" + i.to_s, category.id)
+        end
+      else
+        next
+      end
+    rescue OpenURI::HTTPError => e
+      case rescue_me(e)
+      when 1
+        retry
+      when 2
+        next
+      end
+    end
+  end
+end
+
+def get_recipe_basic_info(url, category_id)
```

Comment:
How is this different from `init`? Here we scrape basic info, but there we scraped basic and in-depth?